app/src/main/res/values/strings.xml

```diff
@@ -26,154 +26,154 @@
-    <string name="desc_saved_servers_list">List of saved Servers</string>
+    <string name="desc_help_btn">Help</string>
+    <string name="desc_discovered_list">List of discovered servers</string>
+    <string name="desc_saved_servers_list">List of saved servers</string>
 
     <!-- Preference -->
     <string name="pref_ui">Interface</string>
     <string name="pref_ui_summary">Look and feel</string>
     <string name="pref_viewer">Viewer</string>
-    <string name="pref_viewer_summary">Picture-in-Picture, Zoom</string>
+    <string name="pref_viewer_summary">Picture-in-picture, zoom</string>
     <string name="pref_input">Input</string>
-    <string name="pref_input_summary">Gestures, Mouse, Virtual keys</string>
+    <string name="pref_input_summary">Gestures, mouse, virtual keys</string>
     <string name="pref_servers">Servers</string>
-    <string name="pref_server_summary">Discovery, Credentials</string>
+    <string name="pref_server_summary">Discovery, credentials</string>
     <string name="pref_tools">Tools</string>
-    <string name="pref_tools_summary">Import/Export, Debugging</string>
+    <string name="pref_tools_summary">Import/export, debugging</string>
     <string name="pref_experimental">Experimental</string>
-    <string name="pref_experimental_summary">Extra goodies!</string>
+    <string name="pref_experimental_summary">Extra goodies</string>
 
     <string name="pref_gesture_action_none">None</string>
-    <string name="pref_gesture_action_left_click">Left Click</string>
-    <string name="pref_gesture_action_middle_click">Middle Click</string>
-    <string name="pref_gesture_action_right_click">Right Click</string>
-    <string name="pref_gesture_action_double_click">Double Click</string>
-    <string name="pref_gesture_action_move_pointer">Move Pointer</string>
-    <string name="pref_gesture_action_pan">Pan Local Frame</string>
-    <string name="pref_gesture_action_remote_scroll">Scroll Remote Content</string>
-    <string name="pref_gesture_action_remote_drag">Drag Remote Content</string>
-    <string name="pref_gesture_action_open_keyboard">Open Keyboard</string>
+    <string name="pref_gesture_action_left_click">Left click</string>
+    <string name="pref_gesture_action_middle_click">Middle click</string>
+    <string name="pref_gesture_action_right_click">Right click</string>
+    <string name="pref_gesture_action_double_click">Double click</string>
+    <string name="pref_gesture_action_move_pointer">Move pointer</string>
+    <string name="pref_gesture_action_pan">Pan local frame</string>
+    <string name="pref_gesture_action_remote_scroll">Scroll remote content</string>
+    <string name="pref_gesture_action_remote_drag">Drag remote content</string>
+    <string name="pref_gesture_action_open_keyboard">Open keyboard</string>
 
     <string name="pref_theme">Theme</string>
-    <string name="pref_theme_option_system">Follow System Theme</string>
+    <string name="pref_theme_option_system">System</string>
```

Comment:
Debatable, but it is generally moving to this.

Reply:
Lets keep the original, it emphasizes the dynamic nature of this option.